notes/stories/literature/novel-various.st.txt

```diff
@@ -67,6 +67,71 @@ cryptid [The party spied a herd of supposedly extinct, prehistoric species of ma
+human vs. captivity [The trio of Professor Aronnax, Conseil, and Ned Land refused to accept that it was their shared fate to remain on board the Nautilus permanently. Although Aronnax felt conflicted because being a prisoner on the Nautilus allowed him to conduct advanced marine research.]
+bird in a gilded cage [Professor Aronnax was reluctant to leave his underwater prison on account of the unique opportunity it afforded him to conduct advanced marine research.]
+the thirst for knowledge [Professor Aronnax toured the ocean depths on a quest for scientific knowledge. Captain Nemo was partially motivated by scientific curiosity.]
+master and servant [Professor Aronnax was accompanied by his loyal man-servant, Conseil, throughout the course of the story.]
+misanthropy [Captain Nemo sought refuge from terrestrial civilization, which he saw as hopelessly flawed, beneath the waves. Indeed, Aronnax wrote of Nemo that "It was no common misanthropy which had shut Captain Nemo and his companions within the Nautilus, but a hatred, either monstrous or sublime, which time could never weaken."]
+the desire for vengeance [It was revealed that Captain Nemo was motivated by a desire to avenge the deaths of his wife and children.]
+
+:: Minor Themes
+anger [Ned Land had a temper tantrum after being confined to a room together with Aronnax and Conseil for an extended period of time.]
+collecting objects [Aronnax was thoroughly impressed with Nemo's collection of painting, which included works by Raphael, da Vince, Holbein, among other illustrious artists.]
+electricity [It was noteworthy that the Nautilus was powered by electric batteries.]
+pleasure in nature [Aronnax described the various natural wonders that he'd beheld under the waves.]
+hunting [Nemo took Aronnax and company on a hunting excursion on the ocean floor. Ned Land hunted birds and small kangaroos on the Papuan island. Ned Land harpooned a giant dugong in waters of the Red Sea.]
+speculative weapon [Nemo and his crewmen used special electric discharge harpoons to hunt prey on the ocean floor.]
+fishing [Nemo's crewmen pulled up some germons and a peculiar species of mackerel in their nets. Aronnax and Conseil dredged the seafloor and hauled up slugs, pearl oysters, a dozen little turtles.]
+facing bloodthirsty savages [Aronnax, Conseil, and Ned Land were attacked by a band of what Aronnax described as "savages" after having trespassed on their island.]
+coping with the death of someone [Nemo buried a fallen crewman in a cemetery on the seafloor. Nemo pointedly grieved for a comrade who'd perished in the fight against the school of giant cuttlefish.]
+human vs. beast [The Nautilus was attacked by a school of giant cuttlefish somewhere near the Bahamas.]
+helping a stranger in need [Nemo risked his life to save an Indian pearl diver from a shark.]
+colonialism [Nemo spoke of India as being an oppressed country.]
```

Comment:
Just fyi, I like to write "British India" or "Raj era India", "colonial India" or something like that to imply which time in history it was

Reply:
I modified it to read "mid 19th century India" (the story is set in the year 1866), since British India (1612–1947), Raj era India (1858–1947), and colonial India (1605–1947) cover pretty big swaths of history.